src/Compilers/VisualBasic/Portable/Binding/Binder_Expressions.vb

```diff
@@ -4361,6 +4375,10 @@ lElseClause:
                     expressionType = DirectCast(expression, UnboundLambda).InferredAnonymousDelegate.Key
                     typeList.AddType(expressionType, RequiredConversion.Any, expression)
 
+                ElseIf expressionKind = BoundKind.TupleLiteral Then
+                    expressionType = DirectCast(expression, BoundTupleLiteral).InferredType
```

Comment:
InferredType can be Nothing now.

---
In reply to: [76157037](https://github.com/dotnet/roslyn/pull/13335#discussion_r76157037) [](ancestors = 76157037)